### DISCUSSION TITLE
Donating Twitter API calls

### DISCUSSION THREAD

**Discussion by developersteve** [danieldevine/bird-elephant#56] [General]
Hi @danieldevine, 

Long time fan of the project! 

So it seems somehow 1 of my long standing (no longer used) twitter developer API projects still has free elevated access (2 million tweets a month). 

Im really not doing anything with them, you are totally welcome to use them if they are of interest.

**Comment by danieldevine** [comment]
Hey Steve,

That's really kind of you, thanks for the offer. My own account is still on elevated (for now)  so not necessary but we'll see what happens in the future. 

Appreciate it anyway.
